checks.d/redisdb.py

```diff
@@ -53,6 +53,8 @@ class Redis(AgentCheck):
 
         # stats
         'latest_fork_usec':             'redis.perf.latest_fork_usec',
+        'bytes_received_per_sec':       'redis.bytes_received_per_sec',
+        'bytes_sent_per_sec':           'redis.bytes_sent_per_sec',
```

Comment:
could you add comments next to these 2 lines as a reminder that these metrics are only available on Azure Redis? :)